lib/ansible/modules/dnf.py

```diff
@@ -19,9 +19,15 @@
 options:
   use_backend:
     description:
-      - By default, this module will select the backend based on the C(ansible_pkg_mgr) fact.
+      - Backend module to use.
     default: "auto"
-    choices: [ auto, yum, yum4, dnf4, dnf5 ]
+    choices:
+        auto: Automatically select the backend based on the C(ansible_facts.pkg_mgr) fact.
+        yum: Alias for V(auto)
```

Comment:
explain that yum is kept for backwards compatibility and that for 'real' yum you should go to older ansible versions.

Reply:
can just have a 'see notes' and explain this in those